documents/code-review.md

```diff
@@ -1,85 +1,66 @@
+- [ ] Make sure there's a `README.md` that describes the project
+- [ ] The readme should contain instructions on running the project locally
 
-- Students are responsible for:
-  - sharing a link to their repo in their cohort's gitter channel prior to 10am, so that all code reviewers (students, mentors, alumni) know where to go
-  - organising which student group will be code reviewed by which other group in the cohort
-  - Clone the repo of the project you're reviewing and run the app from your local machine, to check whether it runs as expected.
-    - Chances are, if you run the app, you will do something slightly different than what the author tried when testing their change. You may discover important cases they missed.
+**Do not work on your own project during code review time.** This is unfair both to the team you should be reviewing, _and_ the team trying to review your changing project.
 
-Mentors of your week will organise code reviews from other mentors/alumni :sparkles:
+### Who reviews what?
 
----
+Each team reviews one other team's project. Graduate mentors and/or FAC alumni will also review.
 
-### 2. Overview of Files in the repo :file_folder:
+Make sure you share a link to your GitHub repo on Slack before code review begins. Try to include any extras the reviewers might need (like API secrets not included in the repo).
 
-Have a quick read through the list of files in the repo.
+### Reviewing
 
-- is there a **README.md** or other documentation?
-  - Understand the purpose of the project through the documentation
-  - Follow any instructions that are in the README.
+Open the repo on GitHub and scan the readme. If the project is deployed online open it and play around. Does the project meet the user stories for that week? Does it demonstrate the learning outcomes for that week? Can you see any obvious bugs or areas to improve?
 
----
+Clone the repo and follow the instructions to run it locally. Does everything work as expected? Often the different environment will highlight issues that the team missed.
 
-### 3. Raise issues :space_invader:
+Open the project in your editor. Does the file structure make sense? Can you you follow the different paths the code might take? Do variables and functions have clear and descriptive names?
 
-- Use the search bar to see whether the issue you want to raise already exists
-  ![Looking at issues](https://files.gitter.im/Jen-Harris/zjN6/image.png)
+### Raise issues
 
-- Raise issues on their repo if you spot any gaps in clarity in the code or documentation.
+Check if the issue you want to raise already exists. Use the search bar if there are lots of issues already. Make sure your issue titles clearly summarise the content. If you're comment on a specific section of code you can [embed it in the issue](https://help.github.com/en/github/managing-your-work-on-github/opening-an-issue-from-code). Link to relevant documentation if you're suggesting something they may not have seen before.
```

Comment:
If you're comment -> If you comment / If you're commenting

Reply:
```suggestion
Check if the issue you want to raise already exists. Use the search bar if there are lots of issues already. Make sure your issue titles clearly summarise the content. If you're commenting on a specific section of code you can [embed it in the issue](https://help.github.com/en/github/managing-your-work-on-github/opening-an-issue-from-code). Link to relevant documentation if you're suggesting something they may not have seen before.
```